docs/docs/workers.md

```diff
@@ -333,21 +332,112 @@ You can use the `-w` option to specify a different worker class to use:
 $ rq worker -w 'path.to.GeventWorker'
 ```
 
+By default, RQ also ships with a few worker classes.
 
-## Strategies for Dequeuing Jobs from Queues
+### SimpleWorker
 
-The default worker considers the order of queues as their priority order.
-That's to say if the supplied queues are `rq worker high low`, the worker will
-prioritize dequeueing jobs from `high` before `low`. To choose a different strategy,
-`rq` provides the `--dequeue-strategy / -ds` option.
+A worker implementation that executes jobs in the same process, without using `fork()`, useful for:
+- Testing and debugging jobs
+- Environments where `fork()` is not available or desired
 
-In certain circumstances, you may want to dequeue jobs in a round robin fashion. For example,
-when you have `q1`,`q2`,`q3`, the 1st dequeued job is taken from `q1`, the 2nd from `q2`,
-the 3rd from `q3`, the 4th from `q1`, the 5th from `q2` and so on.
-To implement this strategy use `-ds round_robin` argument.
+`SimpleWorker` does not provide periodic heartbeats during job execution.
+This means long-running jobs may appear to be "stuck" from monitoring tools' perspective.
+`SimpleWorker` is not recommended for production use unless you fully understand these limitations.
 
-To dequeue jobs from the different queues randomly,  use `-ds random` argument.
+Usage:
+```python
+from rq import SimpleWorker, Queue
+
+queue = Queue('default')
+worker = SimpleWorker([queue])
+worker.work()
+```
+
+Or via CLI:
+```bash
+rq worker -w rq.worker.SimpleWorker
+```
+
+### RoundRobinWorker
+
+The `RoundRobinWorker` dequeues jobs from multiple queues in a round-robin fashion.
+For example, if you have queues `q1`, `q2`, `q3`:
+
+- First job is taken from `q1`
+- Second job from `q2`
+- Third job from `q3`
+- Fourth job from `q1` again
+- And so on...
+
+This provides fair scheduling across queues rather than strict priority ordering.
+
+Usage:
+```python
+from rq import RoundRobinWorker, Queue
+
+queues = [Queue('q1'), Queue('q2'), Queue('q3')]
+worker = RoundRobinWorker(queues)
+worker.work()
+```
+
+Or via CLI:
+```bash
+rq worker -w rq.worker.RoundRobinWorker q1 q2 q3
+```
+
+### RandomWorker
+The `RandomWorker` dequeues jobs from queues randomly rather than in a fixed order.
+This can be useful for load balancing across queues.
+
+Usage:
+```python
+from rq import RandomWorker, Queue
+
+queues = [Queue('q1'), Queue('q2'), Queue('q3')]
+worker = RandomWorker(queues)
+worker.work()
+```
+
+Or via CLI:
+```bash
+rq worker -w rq.worker.RandomWorker q1 q2 q3
+```
+
+### SpawnWorker
+_New in version 2.2.0._
+
+The `SpawnWorker` uses `os.spawn()` instead of `os.fork()` to run jobs. This is useful in environments
+where `fork()` is not available, like Windows or MacOS (`os.fork()` is discouraged).
```

Comment:
I _think_ [fork is being discouraged](https://docs.python.org/3/library/multiprocessing.html) everywhere and spawn will be the default multiprocessing method starting in 3.14:

> The default start method will change away from fork in Python 3.14. Code that requires fork should explicitly specify that via [get_context()](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.get_context) or [set_start_method()](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.set_start_method).